.cbmc-batch/stubs/memset_using_uint64.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * FUNCTION: memset
+ *
+ * Override the version of memset used by CBMC.
+ * This takes advantage of the fact that 64bit operations require fewer array updates,
+ * which can make this version faster than the naive unrolling when used in CBMC.
+ * Benchmark your particular proof to know for sure.
+ */
+
+#include <stddef.h>
+#include <stdint.h>
+
+void *memset_using_uint64_impl(void *dst, int c, size_t n) {
```

Comment:
Header comment here explaining why I'd use this instead of our original implementation. Same for other `using_uint64` functions

Reply:
The comment is actually there, just offset by two lines.  I'll move it down.